src/components/sbb-selection-panel/sbb-selection-panel.tsx

```diff
@@ -127,7 +125,7 @@ export class SbbSelectionPanel implements ComponentInterface {
   }
 
   @Listen('state-change', { passive: true })
-  public onInputChange(event: CustomEvent<RadioButtonStateChange | CheckboxStateChange>): void {
+  public onInputChange(event: CustomEvent<SbbCheckboxStateChange>): void {
```

Comment:
what about the radio button state change?

Reply:
I'll missed it, added.